docs/EN_US/ContainerizedHPCC/ContainerizedMods/ConfigureValues.xml

```diff
@@ -1195,358 +1195,791 @@ thor:
+            trusted clients.</para>
+          </listitem>
+        </itemizedlist></para>
 
-        <para>Cross-origin resource sharing (CORS) is a mechanism for
-        integrating applications in different domains. CORS defines how client
-        web applications in one domain can interact with resources in another
-        domain. You can configure CORS support settings in the ESP section of
-        the values.yaml file as illustrated below:</para>
+        <note>
+          <para>Although an effort was made to standardize on the term TLS
+          throughout this section, there may be references to the older SSL
+          standard it replaced. Any SSL references should be taken as
+          colloquial or legacy nomencalature that actually refers to a TLS
+          implementation.</para>
+        </note>
 
-        <programlisting>esp:
-- name: eclwatch
-  application: eclwatch
-  auth: ldap
-  replicas: 1
-  # The following 'corsAllowed' section is used to configure CORS support
-  #   origin - the origin to support CORS requests from
-  #   headers - the headers to allow for the given origin via CORS
-  #   methods - the HTTP methods to allow for the given origin via CORS
-  #
-  corsAllowed:
-  # origin starting with https will only allow https CORS
-  - origin: https://*.example2.com
-    headers:
-    - "X-Custom-Header"
-    methods:
-    - "GET"
-  # origin starting with http will allow http or https CORS
-  - origin: http://www.example.com
-    headers:
-    - "*"
-    methods:
-    - "GET"
-    - "POST" </programlisting>
-      </sect3>
-    </sect2>
+        <para>The following examples are to exhibit basic functionality in a 
+        testing environment- production deployments may require different
+        settings. The instructions reference paths that are relative to the
+        root of the HPCC-Platform repository- adjust as needed for your
+        environment.
+        </para>
+        
+        <para> A prerequisite to all these examples is installing cert-manager to your
+        Kubernetes cluster:
+        </para>
+
+        <para>
+        Add Jetstack helm repo:
+
+        ```bash
+        helm repo add jetstack https://charts.jetstack.io
+        ```
+
+        Install cert-manager.
+
+        ```bash
+        helm install cert-manager jetstack/cert-manager --set crds.enabled=true --namespace cert-manager --create-namespace
+        ```
+        </para>
+
+        <sect4 id="MutualTLSWithinCluster">
+          <title>Cluster mTLS</title>
+
+          <para>This approach requires creating a root certificate and private key for
+          a local cluster certificate authority, then mounting them as Kubernetes secrets
+          for cert-manager to use.</para>
+
+          <para>Create a root certificate and private key for our local cluster certificate authority with
+          a single openssl call. This call uses a sample openssl config file found in the repo.
```

Comment:
Question: Isn't openssl a proper name? - is it a brand or product name?

Nitpick: I wouldn't truncate repository

Reply:
Yep, adjusted as it's styled on their website.